concepticondata/conceptlists/Koelle-1854-280.tsv

```diff
@@ -267,35 +267,35 @@ Koelle-1854-280-231	231	Poor	Poor	1674	POOR
 Koelle-1854-280-232	232	Straight	Straight	1404	STRAIGHT
 Koelle-1854-280-233	233	Crooked (Bent)	Crooked (Bent)//Crooked (bent)	297	CROOKED
 Koelle-1854-280-233a	233a	Crooked (of men)			
-Koelle-1854-280-234	234	I go	I go		
-Koelle-1854-280-235	235	I come	I come		
-Koelle-1854-280-236	236	I run	I run		
-Koelle-1854-280-237	237	I stop	I stop		
-Koelle-1854-280-238	238	I sit down	I sit down		
-Koelle-1854-280-239	239	I lie down	I lie down		
-Koelle-1854-280-240	240	I breathe	I breathe		
-Koelle-1854-280-241	241	I cough			
-Koelle-1854-280-241a	241a	cough	cough		
-Koelle-1854-280-242	242	I sneeze	I sneeze		
-Koelle-1854-280-243	243	I snore	I snore		
-Koelle-1854-280-244	244	I laugh	I laugh		
-Koelle-1854-280-245	245	I weep	I weep		
-Koelle-1854-280-246	246	I kneel	I kneel		
-Koelle-1854-280-247	247	I dream	I dream		
-Koelle-1854-280-248	248	I sleep	I sleep		
-Koelle-1854-280-249	249	I die	I die		
-Koelle-1854-280-250	250	I fall	I fall		
-Koelle-1854-280-251	251	I rise	I rise		
-Koelle-1854-280-252	252	I speak	I speak		
-Koelle-1854-280-253	253	I hear	I hear		
-Koelle-1854-280-254	254	I beg	I beg		
-Koelle-1854-280-255	255	I bathe (wash myself)	I bathe (wash myself)		
-Koelle-1854-280-256	256	I see	I see		
-Koelle-1854-280-257	257	I take	I take		
-Koelle-1854-280-258	258	I buy	I buy		
-Koelle-1854-280-259	259	I sell	I sell		
-Koelle-1854-280-260	260	I love thee			
-Koelle-1854-280-261	261	I give thee	I give thee		
+Koelle-1854-280-234	234	I go	I go	695	GO
+Koelle-1854-280-235	235	I come	I come	1446	COME
+Koelle-1854-280-236	236	I run	I run	1519	RUN
+Koelle-1854-280-237	237	I stop	I stop	2880	HALT (STOP)
+Koelle-1854-280-238	238	I sit down	I sit down	1649	SIT DOWN
+Koelle-1854-280-239	239	I lie down	I lie down	215	LIE DOWN
+Koelle-1854-280-240	240	I breathe	I breathe	1407	BREATHE
+Koelle-1854-280-241	241	I cough		879	COUGH
+Koelle-1854-280-241a	241a	cough	cough	879	COUGH
+Koelle-1854-280-242	242	I sneeze	I sneeze	1621	SNEEZE
+Koelle-1854-280-243	243	I snore	I snore	1983	SNORE
+Koelle-1854-280-244	244	I laugh	I laugh	1355	LAUGH
+Koelle-1854-280-245	245	I weep	I weep	1839	CRY
+Koelle-1854-280-246	246	I kneel	I kneel	66	KNEEL
+Koelle-1854-280-247	247	I dream	I dream	1920	DREAM (SOMETHING)
+Koelle-1854-280-248	248	I sleep	I sleep	1585	SLEEP
+Koelle-1854-280-249	249	I die	I die	1494	DIE
+Koelle-1854-280-250	250	I fall	I fall	1280	FALL
+Koelle-1854-280-251	251	I rise	I rise	568	RISE (MOVE UPWARDS)
```

Comment:
RISE (MOVE UPWARDS) refers to heavenly bodies (e.g. the sun in the morning), I don't think it's meant here.
GO UP OR RISE (ONESELF) might fit better.
But it's ambiguous.

Reply:
we can unmap it